server/src/internalClusterTest/java/org/opensearch/discovery/MasterDisruptionIT.java

```diff
@@ -91,7 +91,7 @@ public void testMasterNodeGCs() throws Exception {
 
         logger.info("waiting for nodes to de-elect master [{}]", oldClusterManagerNode);
```

Comment:
`master` -> `cluster_manager`

Reply:
Same as above comment. The master word in the log is changed in the other PR https://github.com/opensearch-project/OpenSearch/pull/2521/files#diff-d5423ad6b9197f9d92b8c463df4320e6e2c3ca85768fde53c7e8d01f5945732e.